indico_jacow/client/MultipleAffiliationsSelector.jsx

```diff
@@ -0,0 +1,253 @@
+    type: 'affiliation',
+    index,
+    moveItem: onMove,
+    separateHandle: true,
+  });
+
+  return (
+    <div ref={itemRef} style={style} styleName="affiliation">
+      <Ref innerRef={dragRef}>
+        <div className="icon-drag-indicator" styleName="handle" />
+      </Ref>
+      <Header
+        style={{fontSize: 14}}
+        content={affiliation.meta.name}
+        subheader={getSubheader(affiliation.meta)}
+      />
+      <Icon name="remove" color="red" size="large" styleName="delete" onClick={onDelete} link />
+    </div>
+  );
+};
+
+DraggableAffiliation.propTypes = {
+  affiliation: PropTypes.shape(affiliationShape).isRequired,
+  onDelete: PropTypes.func.isRequired,
+  index: PropTypes.number.isRequired,
+  onMove: PropTypes.func.isRequired,
+};
+
+const MultipleAffiliationsField = ({onChange, value, currentAffiliations}) => {
+  const [_searchResults, setSearchResults] = useState([]);
+  const searchResults = [
+    ...currentAffiliations,
+    ..._searchResults.filter(x => !currentAffiliations.find(y => y.id === x.id)),
+  ].filter(x => !value.find(y => y.meta.id === x.id));
+
+  const affiliationOptions = searchResults.map(res => ({
+    key: res.id,
+    value: res.id,
+    text: res.name,
+    meta: res,
+    content: <Header style={{fontSize: 14}} content={res.name} subheader={getSubheader(res)} />,
+  }));
+
+  const searchChange = async (evt, {searchQuery}) => {
+    if (!searchQuery) {
+      setSearchResults([]);
+      return;
+    }
+    let resp;
+    try {
+      resp = await debounce(() => indicoAxios.get(searchAffiliationURL({q: searchQuery})));
+    } catch (error) {
+      handleAxiosError(error);
+      return;
+    }
+    setSearchResults(camelizeKeys(resp.data));
+  };
+
+  const addItem = (evt, {value: newId}) => {
+    const newValue = searchResults.find(x => x.id === newId);
+    onChange([...value, {id: newValue.id, text: newValue.name, meta: newValue}]);
+    setSearchResults([]);
+  };
+
+  const moveItem = (dragIndex, hoverIndex) => {
+    const result = value.slice();
+    result.splice(hoverIndex, 0, ...result.splice(dragIndex, 1));
+    onChange(result);
+  };
+
+  return (
+    <DndProvider backend={HTML5Backend}>
+      <Segment attached="top">
+        <SortableWrapper accept="affiliation">
+          <List divided relaxed>
+            {value.length > 0 ? (
+              value.map((affiliation, idx) => (
+                <DraggableAffiliation
+                  key={affiliation.id}
+                  index={idx}
+                  onMove={moveItem}
+                  affiliation={affiliation}
+                  onDelete={() => onChange(value.filter((_, i) => i !== idx))}
+                />
+              ))
+            ) : (
+              <Translate>There are no affiliations</Translate>
+            )}
+          </List>
+        </SortableWrapper>
+      </Segment>
+      <Segment attached="bottom">
+        <Dropdown
+          icon="search"
+          placeholder="Add affiliations..."
```

Comment:
Missing i18n (not that it matters for that plugin, but you added i18n markers elsewhere so there we go :P)